Nuake/src/Resource/Project.cpp

```diff
@@ -53,6 +53,12 @@ namespace Nuake
 		projectFile.close();
 	}
 
+	bool Project::FileExist()
+	{
+		struct stat buffer{};   
```

Comment:
I prefer using std::filesystem over stat. It's from c++ 17.

https://en.cppreference.com/w/cpp/filesystem/exists